bag_path: /media/adam/Data/PhD/Data/force_push/heins-ral24/box10_straight_15-24-54/bag_2023-09-30-15-24-55.bag
r_co_o:
- -0.0027220375054552437
- -0.010154241473549522
- 0.00010991204187615569
